community/blog/2022-10 Arcus Templates v1.0 a major release with major features/README.md

```diff
@@ -0,0 +1,91 @@
+# Arcus Templates v1.0: a major release with major features
+The Arcus Templates library is less prone to breaking changes as it does not provide code but starter projects. Because of this, major releases could be unnecessarily postponed. We decided to release Arcus Templates v1.0 as the number of features and changes is far too great for a minor release. Come see it for yourself in this run-down post of the release's highlights.
+
+## Introducing isolated Azure Functions
+Isolated Azure Functions is an alternative way for in-process Azure Functions. It runs the implementation as a separate process - hence, the name 'isolated'. This provides many benefits to the consumer. One of which is more control over the hosting capabilities of the function. For more information on the difference between the two, see [Microsoft's documentation](https://learn.microsoft.com/en-us/azure/azure-functions/dotnet-isolated-process-guide).
+
+This release introduces a project option to configure the functions worker of our Azure Functions project templates. Both in-process and isolated are supported; until Microsoft decides to stop supporting the former. Currently, only the Azure Functions Service Bus topic project template (and the new EventHubs project template) are having this option but rest assured, soon the remaining templates will follow suit.
+
+The biggest difference in isolated projects is that the project will have a `Program` instead of a `Startup` file. The Azure Functions project acts like a regular .NET worker project, in this way. Different from the worker templates is that Arcus only registers the message router because the Azure Functions is the message pump for the Service Bus message. Here is a snippet from the `Program` file that shows how we can make use of the `ConfigureFunctionsWorkerDefaults` extension to register all the necessary messaging functionality.
```

Comment:
```suggestion
The biggest difference in isolated projects is that the project will have a `Program` instead of a `Startup` file. The Azure Functions project acts like a regular .NET worker project, in this way. Different from the worker templates is that Arcus only registers the message router because the Azure Function is the message pump for the Service Bus message. Here is a snippet from the `Program` file that shows how we can make use of the `ConfigureFunctionsWorkerDefaults` extension to register all the necessary messaging functionality.
```